frontend/urls.py

```diff
@@ -4,13 +4,15 @@
 
 from . import views
 
-
+app_name = "gertrude"
```

Comment:
I don't see this variable being used here, am I missing something?

Reply:
Well apparently that's used to define a namespace, to later use it with reverse() x_x

c.f. "polls" in here : https://docs.djangoproject.com/en/2.0/topics/http/urls/#reversing-namespaced-urls

(and I actually had to modify the generic_confirmation lib too :| )